_posts/2023-02-22-solidity-0.8.19-release-announcement.md

```diff
@@ -0,0 +1,47 @@
+date: '2023-02-22'
+author: Solidity Team
+category: Releases
+---
+
+Introducing the newest version of the Solidity Compiler!
+
+We are excited to announce the latest release of the Solidity Compiler, [Solidity v0.8.19](https://github.com/ethereum/solidity/releases/tag/v0.8.19). 
+This latest version includes a range of improvements and it also introduces support defining *operators* on [user defined value types](https://docs.soliditylang.org/en/latest/types.html#user-defined-value-types) (UDVTs)!
+
+### Notable New Features
+These are the features we want to highlight in this release.
+
+#### Operators for user defined value types
+The ability to use operators is meant to bring the UDVTs closer to being as natural to use
+as the built-in types and promote the use of type-safe interfaces.
+
+TODO You can read more on @cameel's latest blogpost
+
+## Full Changelog
+### Language Features
+* TODO
+
+### Compiler Features
+* TODO
+
+### Bugfixes
+* TODO
+
+As always, a big thank you to all contributors who helped make this release possible!
+
+## How to install/upgrade
+To upgrade to the latest version of the Solidity Compiler, simply follow the [installation instructions](https://docs.soliditylang.org/en/v0.8.19/installing-solidity.html) available in our documentation. 
+Our team has made sure to provide detailed and straightforward steps to make the upgrade process as seamless as possible. If you have any questions or run into any issues during the upgrade process, don't hesitate to reach out to our [community Matrix channel](https://matrix.to/#/#ethereum_solidity:gitter.im). 
+
+You can download the new version of Solidity here: [v0.8.19](https://github.com/ethereum/solidity/releases/tag/v0.8.19).
+If you want to build from the source code, do not use the source archives generated automatically by GitHub, instead please use [`solidity_0.8.19.tar.gz`](https://github.com/ethereum/solidity/releases/download/v0.8.18/solidity_0.8.19.tar.gz) and take a look at [our documentation on how to build from source](https://docs.soliditylang.org/en/v0.8.19/installing-solidity.html#building-from-source) if you need guidance.
+
+Please note that for those using Solidity versions below 0.8.0, [there are breaking changes](https://docs.soliditylang.org/en/v0.8.18/080-breaking-changes.html). 
+We highly recommend reviewing the detailed list of breaking changes in our documentation to ensure a smooth upgrade process. Additionally, we encourage all Solidity users to regularly check for updates to stay up-to-date with the latest improvements and optimizations.
+We advise all Solidity developers to upgrade to version 0.8.18 to take advantage of these improvements and optimizations. Our team remains dedicated to continuously enhancing Solidity to benefit the entire Ethereum ecosystem.
+
+Get ready to unleash your creativity and build impactful decentralized applications with the latest version of the Solidity Compiler. Happy coding!
```

Comment:
We have technical audience - I'd prefer a bit less of marketing-ish hype-talk :-).

Reply:
The last release was like that. One sentence will not ruin your street cred don't fret. 😄

Reply:
Yes, I didn't like it then either :-D.

Reply:
Maybe we should reserve this kind of language for the twitter announcment?

Reply:
```suggestion
```
Removed.